DLaB.CrmSvcUtilExtensions/Entity/EntityConstructorsGenerator.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Microsoft.Crm.Services.Utility;
+using System;
+using System.CodeDom;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DLaB.CrmSvcUtilExtensions.Entity
+{
+    internal class EntityConstructorsGenerator : ICustomizeCodeDomService
+    {
+        public void CustomizeCodeDom(CodeCompileUnit codeUnit, IServiceProvider services)
+        {
+            var metadataService = (IMetadataProviderService)services.GetService(typeof(IMetadataProviderService));
+            var metadata = metadataService.LoadMetadata();
```

Comment:
Looks like this isn't used?

Reply:
Ahm... true. Sharp eyes!
Should I remove it?

Reply:
Yes Please